js/understanding-hoisting-the-real-way/understanding-hoisting-the-real-way.md

```diff
@@ -0,0 +1,185 @@
+Hello, I am Shivam Verma. Today we are going to understand one of the most frequently asked questions in interviews. 
```

Comment:
> Today we are going to understand one of the most frequently asked questions in interviews. 

Put Emphasis on the topic here. Example: Today we are going to understand hoisting. It also happens to be one of the most frequently asked questions in interviews. Let's start.